src/js/components/Menu/Menu.js

```diff
@@ -88,6 +88,7 @@ const Menu = forwardRef((props, ref) => {
   // when there's not enough space below DropButton. This state
   // is modified on /Drop/DropContainer.js.
   const [alignControlMirror, setAlignControlMirror] = useState();
+  const shrunk = alignControlMirror === align.top;
```

Comment:
I am wondering about this variable name. Wondering if something like `alignTop` would make more sense?

Reply:
Would `initialAlignTop` be a good name for this since we made this boolean to check if there was an align.top when starting?